ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -117,6 +117,7 @@ type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
     "comment_options" ? ssh_daemon_options_type
     "sshd_path" ? string
+    "validate_config" ? string with match (SELF, '^(yes|no|optional)$')
```

Comment:
OK, I've changed it to a boolean called validate_optional. Disabling the checks completely has been removed.

Our sshd is a component which happens to be managed in the network filesystem, installing another through a package isn't something we would want to do. The config validation is very nice to have after build, but we've done without it previously and it would be easiest to just allow the check to skip at build time.